jvm/src/test/scala/com/nawforce/apexlink/pkg/CachedTest.scala

```diff
@@ -206,7 +206,21 @@ class CachedTest extends AnyFunSuite with TestHelper with BeforeAndAfter {
     FileSystemHelper.run(
       Map(
         "pkg1/Foo.cls" -> "public class Foo extends Bar {}",
-        "pkg2/Foo.cls" -> "public class Foo extends Bar {}"
+        "pkg1/sfdx-project.json" -> """{
+  "packageDirectories": [
+    {"path": ".", "default": true}
+  ],
+  "sfdcLoginUrl": "https://login.salesforce.com",
+  "sourceApiVersion": "48.0"
+}""",
+        "pkg2/Foo.cls" -> "public class Foo extends Bar {}",
+        "pkg2/sfdx-project.json" -> """{
+  "packageDirectories": [
+    {"path": ".", "default": true}
+  ],
+  "sfdcLoginUrl": "https://login.salesforce.com",
+  "sourceApiVersion": "48.0"
+}"""
```

Comment:
Can shorten these test changes if you expose `defaultSFDXProject` from the test helper.

Reply:
Done